src/renderers/wgpu/vbo.rs

```diff
@@ -0,0 +1,59 @@
+use wgpu::util::DeviceExt;
+
+pub enum Vbo<T: Copy> {
```

Comment:
Oh, you had to copy all of this...
Try to see if you can pull my latest improvement on buffers and reuse my `GlBuffer` struct so that it uses the same struct for both, but has one `upload` function for OpenGL and a different one for WGPU.